packages/expo-tracking-transparency/ios/TrackingTransparencyModule.swift

```diff
@@ -25,5 +26,9 @@ public class TrackingTransparencyModule: Module {
         reject: promise.legacyRejecter
       )
     }
+
+    AsyncFunction("getAdvertisingIdAsync") { () -> String in
+      return ASIdentifierManager.shared().advertisingIdentifier.uuidString
```

Comment:
Looks like on both iOS and Android the returned UUID can be with all zeros. I'm wondering if it wouldn't be better to just return `null` in that case. The user will usually have to do one more check whether the id is present, but it increases the awareness that this value may be unavailable in some scenarios. What are your thoughts?

Reply:
Good thought 🙏 
Since it's the same for iOS and Android, I'll do the check on the JS side.